### DISCUSSION TITLE
Anyone successful with scrypted on Windows?

### DISCUSSION THREAD

**Discussion by cheme75** [mrlt8/docker-wyze-bridge#378] [Q&A] [answered]
with so much positive about scrypted, and since I already got docker on my win pc doing ok for this bridge, I figured I'd give scrypted a try. What a cluster - the dev seems oblivious to helping with anything win related and user support has been nearly nil. I see container and it seems to be running but nothing I can do will acess the UI so it basically useless at this point. I deleted the container, will double check yml file and try one more time. Meanwhile someone said to try discord but not win help there either. Hoping someone in these discussions might point to some decent help since my searches come up with nothing good so far.
On a positive note - the lack of support for scrypted makes me even more impressed by @mrlt8 and all the support he has offered!

**Comment by mrlt8** [comment, marked as answer]
I believe this is a limitation of docker on windows as [host mode is not supported](https://docs.docker.com/network/network-tutorial-host/#prerequisites) which the scrypted container [requires](https://github.com/koush/scrypted/wiki/Installation:-Docker-Desktop).

Edit: https://github.com/koush/scrypted/wiki/Installation:-WSL2-Windows

**Reply by cheme75** [reply]
I guess I didn’t understand the win install info clearly, it’s somewhat limited and not really insightful help is posted there. I’ll setup the bridge for the wsl adapter with my lan adapter. and try one more win install. If it fails again I may try docker on my lubuntu netbook for both homebridge and scrypted. Hate to mess around too much with the win docker pc since it’s going ok with this bridge.

**Reply by cheme75** [reply]
@mrlt8 well, gave up on win install, royal pain. Installed docker on my old lubuntu netbook that I setup for hoobs. Fumbled my way through setting up scrypted - didn’t  really understand setting up multiple cams in the rtsp plug-in so that took some trial and error. I’d specify each stream, but somehow the cam display always showed 2 of cam1 and none of cam2 or the doorbell cam. Anyway detected and added until I got all 3 and then added the bridge to HomeKit. It works ok, but definitely not happy I can’t rotate the doorbell or my sideways v2 like I can in ffmpeg in homebridge. So it’s got hksv which should prove more useful in the long run once I figure out the detection stuff, but the homebridge ffmpeg stream looks better being rotated correctly. Anyway, did some searching and found nothing about rotating. If anyone here knows how, please advise!

**Reply by mrlt8** [reply]
Have you tried the ffmpeg plugin `@scrypted/ffmpeg-camera`? I believe it has some extra options to pass custom commands.

**Reply by cheme75** [reply]
Ya, that’s what I’m thinking to try. After a lot of flipping back and forth between each cam setting, I finally got all 3 in both scrypted dashboard and HomeKit. I tried - transpose=1 in the transcode h264 arguments section and checked it on for that cam but it didn’t do anything. Will try ffmpeg plugin later.
